luadox/render/html.py

```diff
@@ -283,10 +288,18 @@ def _render_html(self, topref: TopRef, lines: List[str]) -> Generator[
         root = self._get_root_path()
         head: list[str] = []
 
-        css = self.config.get('project', 'css', fallback=None)
-        if css:
+        css_files = self.config.get('project', 'css', fallback='').split()
```

Comment:
Hm, we will need the same splitlines()/shlex.split() logic here too.  Because get_files() in main.py as well as copy_files_from_config() both need this logic (as discussed elsewhere), and now we need it here too, I think a utility function makes sense, which can be added to utils.py.

Something like:

```python
def files_str_to_list(files: str) -> Generator[str, None, None]:
    for line in files.strip().splitlines():
        yield from shlex.split(line)
```
(`Generator` comes from the typing module.)